### DISCUSSION TITLE
FTP Upload settings and video viewer app ?

### DISCUSSION THREAD

**Discussion by chrbar11** [roleoroleo/yi-hack-Allwinner-v2#629] [General]
Hi,

I run Yi-Hack 0.2.6 on model h30ga with base version 9.0.26*.
Yi App and all the Cloud features are disable, and recording on the SD is enable.

I've set FTP Upload on Yi-Hack, and it works great.

If I disable "FTP Delete (after upload)", and I set "Auto video cleaning" on 20 (% SD free space).
Does it mean that camera is going to record video files on the SD, copy them on FTP, and start to delete older files when SD card will be filled at 80% ?

I'm looking for a nice app to watch records uploaded on FTP.
Do you have any advices?
My FTP storage is on linux server thus I'd prefer an app run as docker.

Thanks a lot for your help :)
Chris

**Comment by roleoroleo** [comment]
> If I disable "FTP Delete (after upload)", and I set "Auto video cleaning" on 20 (% SD free space).
Does it mean that camera is going to record video files on the SD, copy them on FTP, and start to delete older files when SD card will be filled at 80% ?

Yes.

> What do you mean exactly?

**Comment by Giuserver** [comment]
> Hi,
> 
> I run Yi-Hack 0.2.6 on model h30ga with base version 9.0.26*. Yi App and all the Cloud features are disable, and recording on the SD is enable.
> 
> I've set FTP Upload on Yi-Hack, and it works great.
> 
> If I disable "FTP Delete (after upload)", and I set "Auto video cleaning" on 20 (% SD free space). Does it mean that camera is going to record video files on the SD, copy them on FTP, and start to delete older files when SD card will be filled at 80% ?
> 
> I'm looking for a nice app to watch records uploaded on FTP. Do you have any advices? My FTP storage is on linux server thus I'd prefer an app run as docker.
> 
> Thanks a lot for your help :) Chris

Kodi
